handbook/business-operations/README.md

```diff
@@ -171,6 +171,18 @@ When a Fleetie, consultant or advisor requests an update to their personnel deta
 - If required, BizOps also makes changes to other core systems (e.g: creating a new email alias in google workspace; updating details in Carta; etc).
 - The change is now actioned, notify the team member and close the issue.
 
+### Updating compensation
+When Fleet [changes compensation for team members](https://fleetdm.com/handbook/company/communications#compensation-changes), the Business Operations team adheres to the following process:
+- On the first Friday of the month, the Head of BizOps posts in the `#help-classified` channel with the list of team members due for compensation review.
```

Comment:
To note: updating compensation can be a ritual (like fleetiversaries), however that is not the only time compensation is updated. It could be for a role change, location change, or ad-hoc at CEO discretion. Regardless of the reason for change, I wanted to capture the process that would be followed.